script/cibuild-libchromiumcontent-linux

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -e
+set -o pipefail
```

Comment:
You can replace these two `set` lines with `set -o errexit -o pipefail` (arguably all of our shell scripts should start with that).